src/test/resources/mapping/expectedTransformationFields.json

```diff
@@ -1,1163 +1,1147 @@
-       "referenceDataValue": "Resource",
-       "path": "$.item.electronicAccess[?(@.relationshipId=='f5d0068e-6272-458e-8a81-b85e7b9a14aa')].uri",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.electronic.access.uri.version.of.resource",
-       "displayNameKey": "item.electronic.access.uri",
-       "referenceDataValue": "Version of resource",
-       "path": "$.item.electronicAccess[?(@.relationshipId=='3b430592-2e09-4b48-9a0c-0636d66b9fb3')].uri",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.enumeration",
-       "displayNameKey": "item.enumeration",
-       "path": "$.items[*].enumeration",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.holdingsrecordid",
-       "displayNameKey": "item.holdingsRecordId",
-       "path": "$.items[*].holdingsRecordId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.hrid",
-       "displayNameKey": "item.hrid",
-       "path": "$.item.hrid",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.id",
-       "displayNameKey": "item.id",
-       "path": "$.item.id",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.action.note",
-       "displayNameKey": "item.itemNoteTypeId",
-       "referenceDataValue": "Action note",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='0e40884c-3523-4c6d-8187-d578e3d2794e' && (!(@.staffOnly) || @.staffOnly == false))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.binding",
-       "displayNameKey": "item.itemNoteTypeId",
-       "referenceDataValue": "Binding",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='87c450be-2033-41fb-80ba-dd2409883681' && (!(@.staffOnly) || @.staffOnly == false))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.staffonly.action.note",
-       "displayNameKey": "item.itemNoteTypeId.staffOnly",
-       "referenceDataValue": "Action note",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='0e40884c-3523-4c6d-8187-d578e3d2794e' && ((@.staffOnly) && @.staffOnly == true))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.staffonly.binding",
-       "displayNameKey": "item.itemNoteTypeId.staffOnly",
-       "referenceDataValue": "Binding",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='87c450be-2033-41fb-80ba-dd2409883681' && ((@.staffOnly) && @.staffOnly == true))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.materialtypeid.book",
-       "displayNameKey": "item.materialTypeId",
-       "referenceDataValue": "book",
-       "path": "$.items[*].materialTypeId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.materialtypeid.dvd",
-       "displayNameKey": "item.materialTypeId",
-       "referenceDataValue": "dvd",
-       "path": "$.items[*].materialTypeId",
-       "recordType": "ITEM"
-     },
```

Comment:
The removed entities for these contain the `referenceDataValue` and the new one does not.

UI pattern for it now looks like `"item.materialTypeId": "Item - Material type - {value}",` where value is `referenceDataValue`. Should the UI be accommodated as well or the `referenceDataValue` is missed by accident?

Reply:
If you see the description of the corresponding story : https://issues.folio.org/browse/MDEXP-279 , MDEXP-217 was stated incorrectly, so it was a bug, 
and yea good point Viktor, we need to make corresponding changes in UI

Reply:
Makes sense. Since the change is rather small, I will handle it as a part of UIDEXP-92 if you don't mind.